examples/auditlog/main.tf

```diff
@@ -0,0 +1,21 @@
+terraform {
+  required_version = ">= 0.13.0"
+  required_providers {
+    oasis = {
+      source = "arangodb.com/managed/oasis"
+      version = ">=1.5.1"
+    }
+  }
+}
+
+provider "oasis" {
+  api_key_id     = "" // API Key ID generated in Oasis platform
+  api_key_secret = "" // API Key Secret generated in Oasis platform
+}
+
+// Terraform created auditlog
+resource "oasis_auditlog" "oasis_test_auditlog" {
```

Comment:
IMHO auditlog should be a subresource of organization, as its organization field can't be changed/updated. What do you think @ewoutp @Robert-Stam ?

Reply:
Nevermind.